src/sympc/tensor/replicatedshare_tensor.py

```diff
@@ -209,6 +231,62 @@ def ne(self, y):
 
         """
 
+    @staticmethod
+    def reconstruct(share_ptrs: List["ShareTensor"], get_shares=False):
+        """Reconstruct value from shares or return shares.
+
+        Args:
+            share_ptrs (List[RSTensorPointers]): List of RSTensor pointers.
+            get_shares (boolean): Return reconstructed values or just shares.
+
+        Returns:
+            reconstructed_value (torch.Tensor): Reconstructed value.
+        """
+        shares1 = share_ptrs[0].get_shares()[0].get()
+        shares2 = share_ptrs[1].get_shares().get()
+
+        if get_shares:
+            return [shares1] + shares2
+
+        return shares1 + sum(shares2)
+
+    @staticmethod
+    def distribute_shares(shares: List[ShareTensor], session: Session):
+        """Distribute a list of shares.
+
+        Args:
+            shares (List[ShareTensor): list of shares to distribute.
+            session (Session): Session.
+
+        Returns:
+            List of ShareTensorPointers.
+
+        """
+        parties = session.parties
+
+        nshares = len(parties) - 1
+
```

Comment:
I think `session_uuid` should also mapped to created RSTensor,similar like what we did for ShareTensor.

Reply:
Will do so.

Reply:
Done.